### ISSUE TITLE
Update Bootstrap from v3 to v4

### ISSUE THREAD

**Issue by bdolor** [BCcampus/eypd#287]
Includes many improvements like dismissible popovers https://getbootstrap.com/docs/4.0/components/popovers/#dismiss-on-next-click

**Comment by bdolor** [comment]
In light of the recently disclosed vulnerability, this ticket needs to be prioritized and addressed: 

- https://nvd.nist.gov/vuln/detail/CVE-2018-14041 
- https://blog.getbootstrap.com/2018/07/12/bootstrap-4-1-2/

**Comment by kathreenriel** [comment]
@bdolor Not sure if this was updated from V3 to V4

**Comment by bdolor** [comment]
we were able to apply the upgrade to tlpd but has yest to be applied to eypd. In light of it being a security concern, I would advise making it a priority

**Comment by kathreenriel** [comment]
Security vulnerability XSS in a framework we use (twbs/bootstrap) 

<img width="471" alt="screen shot 2018-11-01 at 3 01 33 pm" src="https://user-images.githubusercontent.com/16693738/47882558-24d62d00-dde7-11e8-9038-1171c46c5905.png">

Link to vulnerability description
https://nvd.nist.gov/vuln/detail/CVE-2018-14041

Fix is to update bootstrap to the latest 4.1.3 https://blog.getbootstrap.com/2018/07/12/bootstrap-4-1-2/

**Comment by bdolor** [comment]
work done on the TLPD project will be relevant, captured in https://github.com/BCcampus/tlpd/issues/84 and in PR's https://github.com/BCcampus/tlpd/pull/92 and https://github.com/BCcampus/tlpd/issues/93

**Comment by brettjforsyth** [comment]
BS4 will run with the default jQuery of WP but yarn throws errors if jQuery is not in its dependancy list. I added WPs jQuery as a devdependancy (I think this might also solve #93). BS4 also moves to a flexbox based layout system. For speed I didn't attempt to rework the whole site to the new BS4 layout mode but overrode the .row CSS for BS4 to make it work like the old BS. I checked all the layouts and things seemed fine but a double check would be good.

BS4 also switches over to 100% relative units. With the fonts moving to relative means that text might shift a bit from what is currently on the site. 

Questions:
1. Is moving to BS4 default flexbox layouts something that is required? 
2. Is there any desire to move BS4s scss into the project?
3. What is generating the dynamic.css file? I couldn't figure it out but it has a lot of overrides etc that could be an issue if the answer to the first question is yes.
